core/examples/driver/main.cpp

```diff
@@ -75,12 +75,20 @@ int main(int /*argc*/, char** /*argv*/)
 {
   char cmd[256];
 
-  std::vector<std::string> availableModules = GetExampleModules();
+  FrameworkFactory factory;
+  Framework* framework = factory.newFramework(std::map<std::string, std::string>());
+  framework->init();
+  framework->Start();
```

Comment:
The `init()` and `Start()` functions are defined in OSGi and from what I can tell, its to facilitate the Start Level Service Specification within OSGi. 

Since CppMicroServices doesn't support the Start Level Service specification, I don't see a compelling reason to keep `init()`. I will remove it.